ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -514,7 +515,12 @@ - (void)requestTokenByCode:(NSString *)code
     {
         [request_data setValue:_requestParams.scope forKey:OAUTH2_SCOPE];
     }
-    
+
+    NSString* enrollId = [ADEnrollmentGateway enrollmentIDForHomeUserID:nil userID:(_requestParams.identifier ? _requestParams.identifier.userId : nil)];
```

Comment:
nit: Doesn’t calling [ADEnrollmentGateway enrollmentIDForHomeUserID:nil userID: _requestParams.identifier.userId] have the same effect? :)